plugins/modules/elb_target_group.py

```diff
@@ -102,9 +102,23 @@
       - The time period, in seconds, during which requests from a client should be routed to the same target. After this time period expires, the load
         balancer-generated cookie is considered stale. The range is 1 second to 1 week (604800 seconds).
     type: int
+  stickiness_app_cookie_duration:
+    description:
+      - The time period, in seconds, during which requests from a client
+        should be routed to the same target. After this time period expires,
+        the application-generated cookie is considered stale. The range is 1 second to 1 week (604800 seconds).
+    type: int
+    version_added: 1.5.0
+  stickiness_app_cookie_name:
+    description:
+      - The name of the application cookie. Required if C(stickiness_type) is
+        C(app_cookie).
```

Comment:
```suggestion
      - The name of the application cookie. Required if I(stickiness_type=app_cookie).
```

Just a niggle I'll merge, but we use `I(...)` for parameters and `C(...)` for values.